daemon/container.go

```diff
@@ -896,21 +896,21 @@ func (container *Container) Copy(resource string) (io.ReadCloser, error) {
 		return nil, err
 	}
 
-	basePath, err := container.getResourcePath(resource)
-	if err != nil {
+	if err := container.mountVolumes(); err != nil {
 		container.Unmount()
```

Comment:
Yea in the happy case. I was just thinking for the error cases. With the assumption, of course, that extra unmounts would be no-ops.  No biggie.